### PR TITLE
chore: add compatibility matrix

### PR DESCRIPTION
**Pull request by Dudeplayz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,23 @@ The current Hilla support has some known limitations:
...
+|  [![Maven Central](https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&versionPrefix=2)](https://central.sonatype.com/artifact/com.github.mcollovati/quarkus-hilla)   | ![Static Badge](https://img.shields.io/badge/QUARKUS-v3.1%2B-blue?style=for-the-badge&logo=Quarkus)  | ![Static Badge](https://img.shields.io/badge/HILLA-v2.1%2B-blue?style=for-the-badge&logo=Vaadin) | ![Static Badge](https://img.shields.io/badge/FLOW-v24.1%2B-blue?style=for-the-badge&logo=Vaadin) |
+| [![Maven Central](https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&versionPrefix=1)](https://central.sonatype.com/artifact/com.github.mcollovati/quarkus-hillchio) | ![Static Badge](https://img.shields.io/badge/QUARKUS-v2.16%2B-blue?style=for-the-badge&logo=Quarkus) | ![Static Badge](https://img.shields.io/badge/HILLA-v1.3%2B-blue?style=for-the-badge&logo=Vaadin) | ![Static Badge](https://img.shields.io/badge/FLOW-v23.3%2B-blue?style=for-the-badge&logo=Vaadin) |
```
Comment (mcollovati) [inline README.md:51]: typo?
```suggestion
| [![Maven Central](https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&versionPrefix=1)](https://central.sonatype.com/artifact/com.github.mcollovati/quarkus-hilla) | ![Static Badge](https://img.shields.io/badge/QUARKUS-v2.16%2B-blue?style=for-the-badge&logo=Quarkus) | ![Static Badge](https://img.shields.io/badge/HILLA-v1.3%2B-blue?style=for-the-badge&logo=Vaadin) | ![Static Badge](https://img.shields.io/badge/FLOW-v23.3%2B-blue?style=for-the-badge&logo=Vaadin) |
```

Comment (mcollovati) [inline README.md:51]: It would be good to replace `[Static Badge]` with the correct alternative text, e.g. for Hilla badge `[Hilla 2.1+]`

File: README.md
Comment (mcollovati) [inline README.md:1]: I noticed that images without links are rendered as links anyway.
I wonder if it is better to use the `<picture>` plug ``

```
<picture><img alt="Hilla 2.1+" src="https://img.shields.io/badge/HILLA-v2.1%2B-blue?style=for-the-badge&logo=Vaadin"></picture>
```

<picture>Hilla 2.1+</picture>

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

**Review by mcollovati — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Dudeplayz) [reply README.md:51]: Yes, good catch!

File: README.md
Comment (Dudeplayz) [reply README.md:1]: Applied.

**Review by mcollovati — APPROVED** [review]

**Pull request merged**